apps/judicial-system/backend/src/app/modules/defendant/models/defendant.model.ts

```diff
@@ -58,6 +60,21 @@ export class Defendant extends Model {
     )
   }
 
+  static async sentToPoliceAdminDate(
+    defendantId: string,
+    caseId: string,
+  ): Promise<Date | undefined> {
+    const dateLog = await DefendantEventLog.findOne({
+      where: {
```

Comment:
Please do not query the database here. Include the defendant event log in the defendant model and the simply search the list of events from there.